dolphin/drivers/dellemc_storage/vmax.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import PyU4V
+from PyU4V import U4VConn
+
+from oslo_log import log
+from dolphin.drivers import driver
+
+LOG = log.getLogger(__name__)
+
+class VMAXStorageDriver(driver.StorageDriver):
+    """VMAXStorageDriver implement the DELL EMC Storage driver,
+    """
+    # Storage_id's that this driver manages
+    storage_ids = []
+
+    def __init__(self, storage_id):
+        super().__init__(storage_id)
+        self.storage_id = storage_id
+
+    @staticmethod
+    def get_storage_registry():
+        register_info = {
+            "server_ip": "192.168.20.158",
+            "port": 8443,
+            "username": "unisphere-user",
+            "password": "secret-pass",
+            "u4v_version":'90',
+            "verify": "unisphere-ssl-cakey-file"
+        }
+        LOG.info('get_storage_registry()')
+        return register_info
+
+    def register_storage(self, context, register_info):
+        try:
+            u4v_version = register_info['u4v_version']
```

Comment:
Please refer the parameters of  the our storages model.
You may take look of API spec
https://app.swaggerhub.com/apis/Pravin_Test_Hotpot/RetreivalManager/1.0.0#free

Reply:
Modified as per the db model, Done.